FlowMagic-SDKTests/ScreenFlowTests/ScreenFlowTests.swift

```diff
@@ -0,0 +1,179 @@
+////
+////  ScreenFlowTests.swift
+////  FlowMagic-SDKTests
+////
+////  Created by Kedar Dhere on 3/24/23.
+////
+//
+import XCTest
+import SwiftUI
+@testable import FlowMagic_SDK
+
+func propertiesAreEqual(_ lhs: Any, _ rhs: Any) -> Bool {
+    let lhsMirror = Mirror(reflecting: lhs)
+    let rhsMirror = Mirror(reflecting: rhs)
+
+    for (lhsChild, rhsChild) in zip(lhsMirror.children, rhsMirror.children) {
+        guard "\(lhsChild.value)" == "\(rhsChild.value)" else { return false }
+    }
+    return true
+}
+
+class ScreenFlowTests: XCTestCase {
+     // Test Register Screen and Get Screen
+    func testRegisterScreens() {
+        // Given
+        let mockScreenFlowProvider = MockScreenProvider()
+        let screenName = "Home"
+        let portNames = ["Login", "SignUp"]
+        let view = AnyView(Home())
+
+        // When
+        mockScreenFlowProvider.registerScreen(screenName: screenName, portNames: portNames, view: view)
```

Comment:
You are testing a mock class?  Why?

(Same question elsewhere.)

Reply:
I've fixed this.